src/core/client/framework/lib/network/extractGraphQLError.ts

```diff
@@ -12,19 +12,48 @@ function isGraphQLErrorWithExtension(
   return Boolean(err && err.extensions);
 }
 
+class MultiError implements Error {
+  public traceIDs: string[];
+
+  public name: string;
+  public message: string;
+
+  constructor(errors: GraphQLResponseErrors) {
+    this.traceIDs = [];
+    this.name = "GraphQLResponseError";
+    this.message = "";
+
+    const messages: string[] = [];
+
+    errors.forEach((e: { message: string }) => {
+      if (e) {
+        const traceID = (e as any).traceID as string;
+
+        if (traceID && !this.traceIDs.includes(traceID)) {
+          this.traceIDs.push(traceID);
+        }
+
+        messages.push(e.message);
+      }
+    });
+
+    this.message = messages.join(", ");
+  }
+}
+
 export default function extractGraphQLError(
   errors: GraphQLResponseErrors
 ): Error | null {
   if (errors.length > 1) {
     // Multiple errors are GraphQL errors.
     // TODO: (cvle) Is this assumption correct?
-    return null;
+    return new MultiError(errors);
   }
   const err = errors[0];
   if (!isGraphQLErrorWithExtension(err)) {
     // No extensions == GraphQL error.
     // TODO: (cvle) harmonize with server.
-    return null;
+    return new MultiError(errors);
   }
```

Comment:
That's odd, I was easily able to make these happen with a simple error on a GraphQL resolver server side. Particularly common in dev builds.

Reply:
I removed it and things keep on chugging, going to keep it this way until I find a reason to re-add a `MultiError` or similar.

Reply:
> That's odd, I was easily able to make these happen with a simple error on a GraphQL resolver server side. Particularly common in dev builds.

Oh if you can provide an example, it would be super helpful!

Reply:
You were right, `MultiError` does exist 😱 I missed that. We can sync about this later.